### ISSUE TITLE
Home App getting 'No Response' back from HomeBridge/Nest

### ISSUE THREAD

**Issue by mewejo** [KraigM/homebridge-nest#75]
Hi,

I'm trying to setup Nest with Homebridge on a Pi, working on with Wemo so I know the bridge is working.

If I try and change the temperature by the slider, it seems to set it, get no response back, and then seems to set it back to what it was before:

`[9/16/2016, 6:09:57 PM] Target temperature for Living Room Thermostat is: 30 C
[9/16/2016, 6:09:57 PM] Target temperature for Living Room Thermostat is: 19 C`

The same sort of thing happens with Siri, he says that he can't do that when asking him to set the temp to X degrees.

Any ideas?

Thank you,

Josh

**Comment by neuhausf** [comment]
+1 Same thing here.

**Comment by philipkocanda** [comment]
Noticed this too. Looks like this happens when my Nest is set to away mode, looking into a fix.

**Comment by simonrb2000** [comment]
Same behaviour when away mode. I have to take it out of away mode before setting temp

**Comment by philipkocanda** [comment]
Wrote a fix! :)

If you want to use it before the pull request is merged, you can use this command to install it:

```
npm install https://github.com/philipkocanda/homebridge-nest.git#fix-away -g
```

**Comment by simonrb2000** [comment]
Wicked - thank you

**Comment by simonrb2000** [comment]
Got this installed and good news is it works, however I have to ask for temp to be set twice. Not sure if your having this issue philipkocanda? 

If its in away mode and I say "Set Thermostat to 21 degrees", all it will do is bring it out of away mode and not set the temperature. I have to then say again "Set Thermostat to 21 degrees" to actually get it working.

**Comment by philipkocanda** [comment]
Yes I noticed this as well. I've tried to fix this yesterday, but couldn't get it to work. I'll look into it some more but I don't know when I'll have time again.

**Comment by simonrb2000** [comment]
At least its not my setup 👍  

It's no problem, just explaining it to the wife she has to say it twice when she is out. Then I get the "but why?" lol!!!! 

Its better than it was anyways so thank you!! :)

**Comment by philipkocanda** [comment]
Lol, sorry about that, that's pretty annoying.

**Comment by simonrb2000** [comment]
It's no real problem philipkocanda

Grateful for the current fix, at least it can be set now whilst away now.. :D
